lightning-background-processor/src/lib.rs

```diff
@@ -79,6 +79,11 @@ const PING_TIMER: u64 = 1;
 /// Prune the network graph of stale entries hourly.
 const NETWORK_PRUNE_TIMER: u64 = 60 * 60;
 
+#[cfg(all(not(test), debug_assertions))]
+const SCORER_PERSIST_TIMER: u64 = 30;
+#[cfg(test)]
+const SCORER_PERSIST_TIMER: u64 = 1;
+
```

Comment:
When you squash commits, could you make sure this and related changes remain in a separate commit with an appropriate commit message?